run/src/main/kotlin/com/cosmotech/run/service/RunServiceImpl.kt

```diff
@@ -324,8 +326,7 @@ class RunServiceImpl(
     val run = getRun(organizationId, workspaceId, runnerId, runId)
     run.hasPermission(PERMISSION_WRITE)
 
-    if (sendRunDataRequest.data!!.isEmpty())
-        throw IllegalArgumentException("Data field cannot be empty")
+    require(sendRunDataRequest.data!!.isNotEmpty()) { "Data field cannot be empty" }
```

Comment:
Should `sendRunDataRequest.data` have been made required in the recent openapi refacto? (Same for `id` just below). 
* If so, maybe this is better done in another PR since this is not the focus of these changes;
* If not, do we want to handle the data-null using `!isNullOrEmpty()` in order to use the error message also for the null case.